README.md

```diff
@@ -1 +1,83 @@
-Placeholder for Checkstyle Regression Tool readme.
+# Regression-tool
+
+Regression-tool is an automation tool to do regression testing based on proposed patch(Pull Request) 
+of checkstyle project.

```

Comment:
Please also expand how we do this testing.
We analyze git, create configurations at runtime to run and look for differences/changes.